app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -431,6 +437,12 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
     }
 
     private fun onNimbusStartupAndUpdate() {
+        // When Nimbus has successfully started up, we can apply our engine settings experiment.
+        // Any previous value that was set on the engine will be overridden from those set in
+        // Core.Engine.DefaultSettings.
+        components.core.engine.settings.trackingProtectionPolicy =
+            components.core.trackingProtectionPolicyFactory.createTrackingProtectionPolicy()
```

Comment:
This is the fix!

Reply:
We should note here that this also potentially affects anything else that uses the `engine-settings`. e.g. tab prioritization. Is there a bug for this?

Reply:
> We should note here that this also potentially affects anything else that uses the `engine-settings`. e.g. tab prioritization. Is there a bug for this?

Documented in the line above. 👍

Reply:
That sounds good! Specifically we should investigate solutions to initialize Nimbus before the engine. We tried this yesterday and gained some insights into why this will be challenging. It's all to easy to introduce a dependency cycle between Nimbus and the engine if we invert initialization ordering, so we should work through this in detail. Can you create ticket (or should we start with a doc?) to discuss this?